sopel/modules/meetbot.py

```diff
@@ -191,17 +217,20 @@ def startmeeting(bot, trigger):
             bot.say("Can't create log directory for this channel, meeting not started!")
             meetings_dict[trigger.sender] = Ddict(dict)
             raise
-            return
     # Okay, meeting started!
     logplain('Meeting started by ' + trigger.nick.lower(), trigger.sender)
     logHTML_start(trigger.sender)
     meeting_actions[trigger.sender] = []
-    bot.say(bold('Meeting started!') + ' use .action, .agreed, .info, '
-            '.chairs, .subject and .comments to control the meeting. to end '
-            'the meeting, type .endmeeting')
-    bot.say('Users without speaking permission can use .comment ' +
-            trigger.sender + ' followed by their comment in a PM with me to '
-            'vocalize themselves.')
+    bot.say(
+        bold('Meeting started!') + ' use .action, .agreed, .info, .link, '
+        '.chairs, .subject and .comments to control the meeting. To end '
+        'the meeting, type .endmeeting'
+    )
```

Comment:
This and other responses that give usage examples should _really_ use `bot.config.core.help_prefix` instead of hard-coded `.`s everywhere. Since you're tweaking/fixing anyway, it would be 150% amazing if you changed all of those!